tests/facts/windows_aliases/windows_aliases.json

```diff
@@ -0,0 +1,5659 @@
+{
+    "command": "Get-Alias | Format-List -Property *",
+    "output": [
+        "",
+        "",
```

Comment:
Please could we trim this down to a readable number of lines! Only need a few varied ones to test it properly.

Reply:
Ha! Found bug. Thanks.